vnext/Microsoft.ReactNative/Utils/ResourceBrushUtils.cpp

```diff
@@ -44,6 +43,33 @@ void UpdateResourceBrush(
   }
 }
 
+void UpdateToggleResourceBrush(
```

Comment:
this function doesn't seem specific to toggle switches though, so the name is a little misleading
shouldn't this be the behavior for all controls (i.e. shouldn't this logic just go into UpdateResourceBrush ?

Reply:
I originally did have this in UpdateResourceBrush but for the other components to use it, similarly to ToggleSwitch they would have to go through their props and change the ones that had set colors to empty solid color brushes. 

_Note: I could move apply this logic to all controls if there was a way to check if a brush was a System brush or not. I couldn't find a way to do this but let me know if there's a solution to this!_